odl/discr/grid.py

```diff
@@ -34,26 +34,21 @@
 
 # ODL imports
 from odl.set.domain import IntervalProd
-from odl.set.sets import Set, Integers
+from odl.set.sets import Set
 from odl.util.utility import array1d_repr, array1d_str
 
 
-__all__ = ('TensorGrid', 'RegularGrid', 'uniform_sampling')
+__all__ = ('TensorGrid', 'RegularGrid', 'uniform_sampling_fromintv',
+           'uniform_sampling')
 
 
-def sparse_meshgrid(*x, **kwargs):
-    """Make a sparse meshgrid with C- or F-contiguous arrays.
+def sparse_meshgrid(*x):
+    """Make a sparse `meshgrid` by adding empty dimensions.
 
     Parameters
     ----------
-    x1,...,xN : array-like
+    x1,...,xN : `array-like`
         Input arrays to turn into sparse meshgrid vectors
-    squeeze : `bool`
```

Comment:
I guess it will go away with the next merge/rebase. But actually I also thought the upstream fix was included.